cli/README.md

```diff
@@ -299,4 +346,6 @@ The history will display all attributes about each release (e.g. label, mandator
 
 Additionally, the history displays the install metrics for each release. You can view the details about how to interpret the metric data in the documentation for the `deployment ls` command above.
 
+By default, the history doesn't display the author of each release, but if you are collaborating on an app with other developers, and want to view who released each update, you can pass the additional `--displayAuthor` (or `-a`) flag to the history command.
```

Comment:
Note that this doc assumes we will go with the name `--displayAuthor` as I proposed in the other PR. If we stick with `--displayReleasedBy` (ugh!), I can change this to reflect that.